src/views/Leads/Table/BulkActions/index.js

```diff
@@ -0,0 +1,153 @@
+import PropTypes from 'prop-types';
+import React, { useMemo, useCallback } from 'react';
+import { connect } from 'react-redux';
+import PrimaryButton from '#rsca/Button/PrimaryButton';
+import DangerConfirmButton from '#rsca/ConfirmButton/DangerConfirmButton';
+import _ts from '#ts';
+import {
+    isDefined,
+} from '@togglecorp/fujs';
+
+import {
+    notifyOnFailure,
+    notifyOnFatal,
+} from '#utils/requestNotify';
+
+import {
+    RequestClient,
+    methods,
+} from '#request';
+
+import {
+    leadsForProjectTableViewSelector,
+    patchLeadAction,
+    removeBulkLeadAction,
+} from '#redux';
+
+import notify from '#notify';
+
+const mapStateToProps = state => ({
+    leads: leadsForProjectTableViewSelector(state),
+});
+
+const mapDispatchToProps = dispatch => ({
+    patchLead: params => dispatch(patchLeadAction(params)),
+    removeBulkLead: params => dispatch(removeBulkLeadAction(params)),
+});
+
+const requestOptions = {
+    bulkLeadDeleteRequest: {
+        url: '/leads/bulk-delete/',
+        method: methods.POST,
+        body: ({ params: {
+            project,
+            leadIds,
+        } }) => ({
+            project,
+            ids: leadIds.join(','), // For comma separated ids
```

Comment:
We should send an array instead!

Reply:
I have been asked to send the data in this format.
`data: {project: PROJECT-ID, ids: <comma separated lead ids>}`

Reply:
I mean the api should instead support list of integers